docs/data/migration/migration-tree-view-v6/migration-tree-view-v6.md

```diff
@@ -25,3 +25,51 @@ In `package.json`, change the version of the tree view package to `next`.
 
 Since `v7` is a major release, it contains changes that affect the public API.
 These changes were done for consistency, improved stability and to make room for new features.
+
+### ✅ Use `SimpleTreeView` instead of `TreeView`
```

Comment:
That's what we have in this PR

`RichTreeView` uses `props.items`
`SimpleTreeView` uses JSX children
`TreeView` is a deprecated component that uses `SimpleTreeView` under the hood

There is no breaking change on the component name (I'd have to double check if the PR contain other BCs before removing the label, I don't think it does).
Do you think we should skip this section in the migration guide?

Reply:
My idea was - could we have avoided the introduction of `SimpleTreeView` and just refactored it? 🤔 
If nothing is blocking it, wouldn't you think that keeping the existing (items - JSX) component as is would introduce the least amount of friction?

Reply:
Thats what I did at some point, but @joserodolfofreitas is in favor of not having a "TreeView" component at some point.
I think the reasoning is that it would look like it's the default component, when we would rather have people use "RichTreeView" whenever possible because it can have pro features at some point.

Reply:
Gotcha. Well, those are also sound arguments. 👍 
There is no right or wrong choice here. 👌

Reply:
But if we reconsider, we can change before the launch of the alpha, it's not trivial but clearly not the end of the world either :+1:

Reply:
Indeed. 👍 
You could also argue, that if we are thinking about `RichTreeView` becoming the main component, it could make sense to name it `TreeView` and have the additional `SimpleTreeView` for JSX usage. 🤔

Reply:
>  it could make sense to name it TreeView and have the additional SimpleTreeView for JSX usage

That was my very 1st version of this PR
And here the issue is that it is super breaking for everybody upgrading because the component "TreeView" still exists but no longer refers to the same component.
But I guess that's a renaming we could do in 1 - 2 years like we are doing "SimpleTreeView" to have a smooth migration strategy.

Reply:
Yes, I understand, but isn't that is why we are providing the `SimpleTreeView` codemod? 🤔 
If it's nonbreaking and using `TreeView` is still possible in v7, do we even need the codemod and listed BC?
Maybe we could provide it, but not even list in BC page and not include in preset-safe? 🤔